demisto_sdk/commands/format/update_playbook.py

```diff
@@ -152,9 +156,60 @@ def update_playbook_task_name(self):
                 if task_name:
                     task['task']['name'] = task_name
 
+    def check_for_subplaybook_usages(self, file_path: str, current_playbook_id: str, new_playbook_id: str) -> None:
+        """Check if the current_playbook_id appears in the file's playbook type tasks and change it if needed"""
+        updated_tasks = []
+        # if the changed file is a playbook get it's data
+        if find_type(file_path) in [FileType.PLAYBOOK, FileType.TEST_PLAYBOOK]:
+            playbook_data = get_yaml(file_path)
+            # go through all the tasks
+            for task_id, task_data in playbook_data.get('tasks').items():
+                # if a task is of playbook type
+                if task_data.get('type') == 'playbook':
+                    id_key = 'playbookId' if 'playbookId' in task_data.get('task') else 'playbookName'
+                    # make sure the playbookId or playbookName use the new id and not the old
+                    if task_data.get('task').get(id_key) == current_playbook_id:
+                        playbook_data['tasks'][task_id]['task'][id_key] = new_playbook_id
+                        updated_tasks.append(task_id)
+
+            # if any tasks were changed re-write the playbook
+            if updated_tasks:
+                if self.verbose:
+                    click.echo(f'Found usage of playbook in {file_path} tasks: '
+                               f'{" ".join(updated_tasks)} - Updating playbookId')
+                write_yml(file_path, playbook_data)
+
+    def update_playbook_usages(self) -> None:
+        """Check if the current playbook is used as a sub-playbook in other changed playbooks.
+        Change the playbook's id in the tasks id needed.
+        """
+        current_playbook_id = self.data['id']
+        new_playbook_id = self.data['name']
+
+        # if the id and name are the same - there is no need for this format.
+        if current_playbook_id == new_playbook_id:
+            return
+
+        # gather all the changed files - if the formatted playbook was
+        # modified then any additional playbook changes were changed alongside it -
+        # we would use git to gather all other changed playbooks
+        try:
+            git_util = GitUtil()
+            all_changed_files = git_util.modified_files(include_untracked=True).union(
```

Comment:
lets improve the variable names here to something like: `modified_files`, `added_files` and `modified_files`